docs/debugging.md

```diff
@@ -191,6 +191,6 @@ docker rm -f $(docker ps -aq)
 ```
 2. Drop all DataHub's docker volumes.
 ```
-docker volume rm -f $(docker volume ls -f name=datahub_*  -q)
+docker volume rm -f $(docker volume ls -f name=datahub_  -q)
```

Comment:
name filter in docker requires only part of the volume name. Therefore, using `name=datahub_*` results in an empty list of volumes and thus the setup does not get nuked properly

Reply:
Makes sense.